server.js

```diff
@@ -1,31 +1,22 @@
-/* eslint-disable no-console */
 import express from 'express';
 import morgan from 'morgan';
-import user from './server/routes/user';
-import mentor from './server/routes/mentor';
-import session from './server/routes/session';
-import '@babel/polyfill';
-
+import index from './server/routes/index';
 
 console.log(process.env.NODE_ENV);
 
 
 const app = express();
 app.use(morgan('dev'));
-
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use('/api/v1/', user);
-app.use('/api/v1/', mentor);
-app.use('/api/v1/', session);
+app.use('/', index);
 app.get('/', (req, res) => {
```

Comment:
@gadishimwe , this routing should be implemented in your `index.js` file in the `routes` folder